app/client/views/home/home.html

```diff
@@ -1,48 +1,52 @@
 <div id="home">
     <section id="hero">
-        <div class="hero-content center">
-            <div ng-class="{ 'fade-out': fadeOut }">
-                <div class="logo-header-container ui grid mobile only tablet only">
-                    <div class="logo-header logo-cog">
-                        <img src="../assets/images/logo-cog.svg">
-                    </div>
-                    <div class="logo-header">
-                        <img src="../assets/images/logo-anteater.svg">
+        <div class="landing-page-logo-container">
+            <img class="before-logo" src="../../assets/images/landing_pg_bg_no_logo_viv.png" >
+            <img class="after-logo" src="../../assets/images/landing_pg_bg_viv.png">
+            <div class="hero-content center">
+                    <div ng-class="{ 'fade-out': fadeOut }">
+                        <div class="logo-header-container ui grid mobile only tablet only">
+                            <div class="logo-header logo-cog">
+                                <img src="../assets/images/logo-cog.svg">
+                            </div>
+                            <div class="logo-header">
+                                <img src="../assets/images/logo-anteater.svg">
+                            </div>
+                        </div>
+                        <span class="hero-title">HackUCI 2019</span>
+                        <div class="hero-button-container">
+                            <a href="" target="_blank" class="btn">
```

Comment:
let's not make it a link since it isn't, and the cursor icon that shows up when hovered is misleading